### DISCUSSION TITLE
blog/upgrade-tospring6.1-parameter-name-retention

### DISCUSSION THREAD

**Discussion by giscus** [korECM/tw-blog#17] [Comments]
# blog/upgrade-tospring6.1-parameter-name-retention

Spring 6.1에서 변경된 Parameter Name Retention에 대해서 알아보고 어떻게 대처할 수 있는지 살펴봅니다

https://www.korecmblog.com/blog/upgrade-tospring6.1-parameter-name-retention

**Comment by Hyeon9mak** [comment]
재밌게 읽었습니다! 좋은 글 감사합니다!
